### PR TITLE
Add Expensify

### PR DESCRIPTION
**Pull request by bondydaa**
Adding Expensify.com as we have recently added TOTP 2FA to our site.

Verified build
```
$ bundle exec rake verify
/Users/dbondy/.rbenv/versions/2.5.1/bin/ruby _deployment/tests/verify.rb
<------------ No errors. You're good to go! ------------>
```

Manually generated the webp format:
```
$ cwebp -quiet -preset icon img/finance/expensify.png -o img/finance/expensify.webp
```

and verified locally:
image

Ranked in top 200k of Alexa:
Screen Shot 2020-12-10 at 11 34 58 AM

I could not get the webp.sh script to run properly so could not verify image locally.
```
$ ./_deployment/webp.sh
find: illegal option -- t
usage: find [-H | -L | -P] [-EXdsx] [-f path] path ... [expression]
       find [-H | -L | -P] [-EXdsx] -f path [path ...] [expression]

$ which find
/usr/bin/find
```

Ran on macOS 10.15.7 (19H15)

### CODE REVIEW

**Comment by phallobst** [comment]
Yeah, the `find` options we use are not compatible with the stock MacOS one, it seems. Maybe we need to add `brew install findutils` to our installation instructions in order to have the GNU variant at our disposal..

You can still create the webp icon yourself with `cwebp -quiet -preset icon img/finance/expensify.png -o img/finance/expensify.webp` if you like.

File: _data/finance.yml [hunk 1]
```diff
@@ -147,6 +147,13 @@ websites:
       - totp
     doc: https://support.exactonline.com/community/s/knowledge-base#All-All-HNO-Task-general-security-gen-auth-totpregphndsktpt
 
+  - name: Expensify
+    url: https://www.expensify.com/
+    img: expensify.png
+    tfa:
+      - totp
+    doc: https://community.expensify.com/discussion/7385/how-to-enable-two-factor-authentication-on-your-account
```
Comment (phallobst) [inline _data/finance.yml]:
```suggestion
    doc: https://community.expensify.com/discussion/7385
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Comment by bondydaa** [comment]
thanks for that manual command, I've updated the description with a screenshot and fixed the URL as well.

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]
Thanks for the pull request @bondydaa! 🥳

**Pull request merged**